src/spring/azext_spring/tests/latest/recordings/test_persistent_storage.yaml

```diff
@@ -17,7 +17,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.46.0 azsdk-python-azure-mgmt-storage/21.0.0 Python/3.9.13 (Windows-10-10.0.22621-SP0)
     method: POST
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Storage/storageAccounts/clitest000002/listKeys?api-version=2022-09-01&$expand=kerb
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Storage/storageAccounts/clitest000002/listKeys?api-version=2023-01-01&$expand=kerb
```

Comment:
Could you please recording this test instead of only replacing the api-version?

Reply:
This test is not related to the changes of this PR, changing API version to avoid test failed. Could we record this test in the next PR?

Reply:
OK, sounds good~